packages/react-native/src/primitives/TextField/__tests__/__snapshots__/TextField.spec.tsx.snap

```diff
@@ -123,7 +123,7 @@ exports[`TextField doesn't render the errorMessage if errorMessage prop is undef
     style={
       Object {
         "alignItems": "center",
-        "borderColor": "hsl(210, 10%, 58%)",
+        "borderColor": "hsl(0, 95%, 30%)",
```

Comment:
Kind of confused why the `borderColor` value updated here since this test is not error related

Reply:
It does: https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-native/src/primitives/TextField/__tests__/TextField.spec.tsx#L73C44-L73C44

Reply:
Ohhhh, is `error` set to `true` for this test? Maybe we should update the snapshot title since it reads as not having an `error`:
```
TextField doesn't render the errorMessage if errorMessage prop is undefined
```

Reply:
Yup, will do!

Reply:
FYI I changed the name and git got confused, so the diff for the snapshot looks funky.